docs/source/user_guide/calculation_plugins/epw.rst

```diff
@@ -0,0 +1,73 @@
+EPW
+++
+
+Description
+-----------
+Plugin for the Quantum Espresso epw.x executable.
+
+Supported codes
+---------------
+* tested from epw.x v5.2 onwards.
+
+Inputs
+------
+* **parent_calculation**, two parents calculations are required for EPW: a PW calculation the get the
```

Comment:
I think you copy-pasted this documentation from another calculation class, but there is a lot that is not correct. The calculation does not take a `parent_calculation` but two parent folders.